package.json

```diff
@@ -11,10 +11,24 @@
     "jquery": "^3.1.0",
     "mongoose": "^4.6.0"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "angular-mocks": "^1.5.8",
+    "grunt": "^1.0.1",
+    "grunt-concurrent": "^2.3.1",
+    "grunt-karma": "^2.0.0",
+    "grunt-nodemon": "^0.4.2",
+    "jasmine": "^2.5.2",
+    "jasmine-node": "^1.14.5",
+    "karma": "^1.3.0",
+    "karma-chrome-launcher": "^2.0.0",
+    "karma-jasmine": "^1.0.2",
+    "karma-read-json": "^1.1.0",
+    "minimatch": "^3.0.3",
```

Comment:
This may not be necessary anymore. Try to run the tests without it and make sure. If so, uninstall.